nexus/db-model/src/vmm_state.rs

```diff
@@ -108,7 +110,7 @@ impl From<VmmState> for omicron_common::api::external::InstanceState {
             VmmState::Rebooting => Output::Rebooting,
             VmmState::Migrating => Output::Migrating,
             VmmState::Failed => Output::Failed,
-            VmmState::Destroyed => Output::Destroyed,
+            VmmState::Destroyed | VmmState::SagaUnwound => Output::Destroyed,
```

Comment:
SagaUnwound should map to `Stopped` so that an `instance_view` API call that produces an instance with an unwound VMM will appear to be in the correct external state. (Possibly `VmmState::Destroyed` should also map to `Stopped`, but today we don't ever expect an `active_propolis_id` to point to a destroyed VMM.)

The start saga test that covers this case (`test_ensure_running_unwind`) currently passes anyway because `sis_move_to_starting_undo` erases the active Propolis ID before marking the VMM as destroyed. IIRC that will change when the rest of the instance update saga lands and Nexus begins to enforce the rule that only that saga can set `active_propolis_id` to NULL.